### DISCUSSION TITLE
The Craneware Group: process for questions on CMS pricers

### DISCUSSION THREAD

**Discussion by CraneBarry** [CMSgov/design-system#3261] [General]
Hi CMS!

I am Product Manager at The Craneware Group coordinating efforts with our engineering team.

We are in the process of plugging into the CMS pricing data as described here: https://www.cms.gov/pricersourcecodesoftware

Question: what is best process to get assistance / support associated with these efforts?

For example, currently we are seeking a definition of these described source values:  _{see attached}_
cms-pricer-request-id
cms-dcn
 

Any assistance or support would be greatly appreciated.
[
![CMS_Pricer_request_for_engineering_assistance](https://github.com/user-attachments/assets/b3a2a3c7-5813-49c4-8982-7f937da10d93)
](url)

**Comment by ghost** [comment]
Hi @CraneBarry,

Can you please file a [help desk ticket](https://jira.cms.gov/servicedesk/customer/portal/1)? That will get routed to CMS.gov engineers.
